src/Magnum/OvrIntegration/Test/EnumTest.cpp

```diff
@@ -115,6 +133,26 @@ void EnumTest::errorType() {
     CORRADE_COMPARE(out.str(), "OvrIntegration::ErrorType::(invalid)\n");
 }
 
+void EnumTest::ovrDetectResult() {
+    std::ostringstream out;
+    Debug(&out) << OvrDetectResult::HmdConnected;
+    CORRADE_COMPARE(out.str(), "OvrIntegration::OvrDetectResult::HmdConnected\n");
+
+    out.str("");
+    Debug(&out) << OvrDetectResult(-1);
+    CORRADE_COMPARE(out.str(), "OvrIntegration::OvrDetectResult::(invalid)\n");
```

Comment:
Yeah, that invalid thing is my most frequent source of copy-paste error, so back then I started doing that for all of the many OvrIntegration enums.